core/src/test/resources/com/squarespace/template/plugins/platform/f-variants-select-5.html

```diff
@@ -40,7 +40,7 @@
     <div class="variant-option-title">color&amp;hue: </div>
     <div class="variant-select-wrapper">
       <select aria-label="Select color&hue" data-variant-option-name="color&amp;hue">
-        <option value="">Select color&hue</option>
+        <option value="">Select color&amp;hue</option>
```

Comment:
How it appears on the PDP: 

<img width="507" alt="Screen Shot 2021-11-08 at 4 22 37 PM" src="https://user-images.githubusercontent.com/1840976/140820244-b59c0297-a548-4526-9ad2-d0f5b70acc37.png">